CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+# Contributing to MSC AniMet
+
+We welcome contributions to MSC AniMet, in the form of issues, bug fixes, documentation or
+suggestions for enhancements. This document sets out our guidelines and best
+practices for such contributions.
+
+It's based on the [Contributing to Open Source Projects
+Guide](https://contribution-guide-org.readthedocs.io/) and pygeoapi's own CONTRIBUTING.md document.
+
+MSC AniMet has the following modes of contribution:
+
+- GitHub Pull Requests
+
+
+## Submitting Bugs
+
+### Due Diligence
+
+Before submitting a bug, please do the following:
+
+* Perform __basic troubleshooting__ steps:
+
+    * __Make sure you're on the latest version.__ If you're not on the most
+      recent version, your problem may have been solved already! Upgrading is
+      always the best first step.
+    * [__Search the issue
+      tracker__](https://github.com/ECCC-MSC/msc-animet/issues)
+      to make sure it's not a known issue.
+
+### What to put in your bug report
+
+Make sure your report gets the attention it deserves: bug reports with missing
+information may be ignored or punted back to you, delaying a fix.  The below
+constitutes a bare minimum; more info is almost always better:
+
+* __What operating system are you using?__ Windows (7, 8, 10, 32-bit, 64-bit),
```

Comment:
for AniMet, the web browser and its version are important as well. Let's add it here? @Dukestep

Reply:
Added. Thanks for the feedback.